src/components/AuthorCardEditing.vue

```diff
@@ -207,6 +212,51 @@ export default defineComponent({
             orcidErrors
         }
     },
+    watch: {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        orcid (_oldVal, _newVal) {
+            if (this.orcid.length === 37 && this.orcidErrors.length === 0) {
```

Comment:
I am rusty on the difference between Options API and Composition API, but I think this is Options API way, which we don't want. I think the Compositions API way to do it is to define a [`watchEffect`](https://vuejs.org/api/reactivity-core.html#watcheffect) or `watch` (same page) function inside the `setup`.
This also means that you don't need `this`, because the variables are all in scope.